[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`

SplitInfinity opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63506
* __->__ #63461

**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

*Before*

<img width="931" alt="Captura de Pantalla 2021-08-18 a la(s) 10 47 21 a  m" src="https://user-images.githubusercontent.com/4392003/129948036-a92204be-7175-456e-ae25-df41333579ce.png">

*After*
<img width="881" alt="Captura de Pantalla 2021-08-18 a la(s) 12 32 42 p  m" src="https://user-images.githubusercontent.com/4392003/129960706-f508fc6d-0ea0-4fad-8213-4321399f2f5f.png">



**Fixes**
This commit fixes #56267.

Differential Revision: [D30799703](https://our.internmc.facebook.com/intern/diff/D30799703)

Commit:
[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`

**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

**Fixes**
This commit fixes #56267.

[ghstack-poisoned]

SplitInfinity commented:
I will add screenshots tomorrow morning after the pytorch bot posts a link to the docs rendered using the contents of this commit.

mruberry commented on torch/_torch_docs.py:
```diff
@@ -9335,6 +9335,10 @@ def merge_dicts(*dicts):
 
+.. seealso::
+
+        :func:`torch.transpose`.
```
add a half sentence explaining the difference/similarity

SplitInfinity replied:
Added, will update the screenshots once the preview gets generated.

mruberry commented on torch/_torch_docs.py:
```diff
@@ -9704,6 +9706,10 @@ def merge_dicts(*dicts):
 
+.. seealso::
+
+        :func:`torch.t`.
```
add a half sentence here, too

SplitInfinity replied:
Added, will update the screenshots once the preview gets generated.

Commit:
Update on "[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`"


**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

*Before*

<img width="931" alt="Captura de Pantalla 2021-08-18 a la(s) 10 47 21 a  m" src="https://user-images.githubusercontent.com/4392003/129948036-a92204be-7175-456e-ae25-df41333579ce.png">

*After*
<img width="885" alt="Captura de Pantalla 2021-08-18 a la(s) 10 46 38 a  m" src="https://user-images.githubusercontent.com/4392003/129948051-d3d98dd0-b727-40de-afe5-86158f887a81.png">


**Fixes**
This commit fixes #56267.

[ghstack-poisoned]

mruberry reviewed (approved):
Thanks @SplitInfinity! Small suggested tweak inline for your review

Commit:
Update on "[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`"


**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

*Before*

<img width="931" alt="Captura de Pantalla 2021-08-18 a la(s) 10 47 21 a  m" src="https://user-images.githubusercontent.com/4392003/129948036-a92204be-7175-456e-ae25-df41333579ce.png">

*After*
<img width="881" alt="Captura de Pantalla 2021-08-18 a la(s) 12 32 42 p  m" src="https://user-images.githubusercontent.com/4392003/129960706-f508fc6d-0ea0-4fad-8213-4321399f2f5f.png">



**Fixes**
This commit fixes #56267.

[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30799703).

Commit:
Update on "[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`"


**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

*Before*

<img width="931" alt="Captura de Pantalla 2021-08-18 a la(s) 10 47 21 a  m" src="https://user-images.githubusercontent.com/4392003/129948036-a92204be-7175-456e-ae25-df41333579ce.png">

*After*
<img width="881" alt="Captura de Pantalla 2021-08-18 a la(s) 12 32 42 p  m" src="https://user-images.githubusercontent.com/4392003/129960706-f508fc6d-0ea0-4fad-8213-4321399f2f5f.png">



**Fixes**
This commit fixes #56267.

Differential Revision: [D30799703](https://our.internmc.facebook.com/intern/diff/D30799703)

[ghstack-poisoned]
